cicd/monit_pypi/Dockerfile

```diff
@@ -0,0 +1,51 @@
+ARG BASE_TAG=latest
+FROM registry.cern.ch/cmscrab/crabtaskworker:${BASE_TAG}
+
+# RUN apt update && apt install -y \
+#     vim nano \
+#     && \
+#     apt clean && rm -rf /var/lib/apt/lists/*
+# # the following packages are necessary, but already provided inside the docker image
+# #  libcurl-openssl-dev, libssl-dev \
+
+# ENV WDIR=/data
+# ENV USER=crab3
+# # add new user and switch to user
+# RUN useradd ${USER} &&\
+#     install -o ${USER} -d ${WDIR} &&\
+#     install -o ${USER} -d /home/${USER}
+# USER ${USER}
+
+# pip is already available in the python:3.8 docker image
+# RUN python3.8 -m ensurepip
+RUN python3.8 -m pip install --user pandas 
```

Comment:
Use simply `pip install`, without version.

Reply:
mh, ok, i don't really like calling `pip` as a command directly because it is easy to install the package under the wrong python version, but ok, this is a python docker container, is should have a single python installed